src/components/shared/modals/ProjectToolDrawerModal.tsx

```diff
@@ -286,68 +253,23 @@ export default function ProjectToolDrawerModal({
 
         <Divider />
 
-        {metadata?.archived ? (
-          <section>
-            <h3>
-              <Trans>Unarchive project</Trans>
-            </h3>
-            <p>
-              <Trans>
-                Your project will immediately appear active on the
-                juicebox.money app. Please allow a few days for it to appear in
-                the "active" projects list on the Projects page.
-              </Trans>
-            </p>
-            <Button
-              onClick={() => setArchived(false)}
-              loading={loadingArchive}
-              size="small"
-              type="primary"
-            >
-              <Trans>Unarchive project</Trans>
-            </Button>
-          </section>
+        {setUriTx ? (
+          <ArchiveV1Project setUriTx={setUriTx} />
         ) : (
           <section>
             <h3>
               <Trans>Archive project</Trans>
             </h3>
             <p>
               <Trans>
-                Your project will appear archived, and will not be able to
-                receive payments through the juicebox.money app. You can
-                unarchive a project at any time. Please allow a few days for
-                your project to appear under the "archived" filter on the
-                Projects page.
+                Please contact the Juicebox dev team through our{' '}
```

Comment:
We'll need to do archived eventually right? If so lets create an issue.

Reply:
Done #799